### ISSUE TITLE
Feature Request: long-form only arguments or multi-letter short arguments

### ISSUE THREAD

**Issue by gdevenyi** [kvz/bash3boilerplate#23]
I'd like to add arguments for some features that have the same starting letter:
--walltime-template
--walltime-subject

Right now, short options are limited to a single letter, so it's not clear how I can expose these.

**Comment by zbeekman** [comment]
I'm thinking of changing things so that long options are the default/required and short options are optional... in the mean time, you'll have to do something like -w --walltime-template, -W --walltime-subject, or something like -t --walltime-template, -s --walltime-subject

**Comment by gdevenyi** [comment]
Cool great. I will be using that in the meantime :)

**Comment by kvz** [comment]
> I'm thinking of changing things so that long options are the default/required and short options are optional

💯

**Comment by gdevenyi** [comment]
Been a while here, wondering if any progress or interest in implementing this?

**Comment by zbeekman** [comment]
no... working on backtrace first... not sure if I'll get to this anytime soon
